schemas/com.snowplowanalytics.snowplow/ip_lookups/jsonschema/2-0-0

```diff
@@ -0,0 +1,90 @@
+{
+	"$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+	"description": "Schema for MaxMind GeoIP2 ip lookups enrichment",
+	"self": {
+		"vendor": "com.snowplowanalytics.snowplow",
+		"name": "ip_lookups",
+		"format": "jsonschema",
+		"version": "2-0-0"
+	},
+
+	"type": "object",
+	"properties": {
+		"vendor": {
+			"type": "string",
+            "maxLength": 256
```

Comment:
Whitespace is all messed up in this file (see 6 more instances below).

Reply:
mix-indent ftw...